libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -207,6 +207,11 @@ class YulUtilFunctions
 	/// signature: (array, newLen)
 	std::string resizeArrayFunction(ArrayType const& _type);
 
+	/// @returns the name of a function that zeros all storage array elements from `start_index` to `len` (excluding)
+	/// Assumes that `len` is the array length. Does nothing if `start_index >= len`. Does not modify the stored length.
+	/// signature: (array, len, start_index)
```

Comment:
The actual implementation uses `(array, len, pos)`.

Reply:
I used `startIndex` everywhere now